pil-analyzer/src/evaluator.rs

```diff
@@ -36,6 +35,8 @@ pub fn evaluate_function_call<'a, T: FieldElement, C: Custom>(
     function: Value<'a, T, C>,
     arguments: Vec<Rc<Value<'a, T, C>>>,
     symbols: &impl SymbolLookup<'a, T, C>,
+    // TODO maybe we should also make this return an Rc<Value>.
```

Comment:
I had a quick look at this, but it required too many changes to `Rc` at the interfaces. We can turn this into an issue.

Reply:
ok, please do

Reply:
https://github.com/powdr-labs/powdr/issues/946